x/docs/md/cli/windows.md

```diff
@@ -1,6 +1,10 @@
 ## Windows
 
-Below are installation instructions, using the most common method, the Chocolately Package Manager. For further help and instructions, see the following:
+[There is now a installer for Windows available on GitHub](https://github.com/exercism/windows-installer/releases)
```

Comment:
Let's remove _now_, since most people reading this page won't care that it wasn't available before.